variables.tf

```diff
@@ -107,3 +107,8 @@ variable "deployment_minimum_healthy_percent" {
   description = "The lower limit (as a percentage of `desired_count`) of the number of tasks that must remain running and healthy in a service during a deployment"
   default     = 100
 }
+
+variable "health_check_grace_period_seconds" {
+  description = "Seconds to ignore failing load balancer health checks on newly instantiated tasks to prevent premature shutdown, up to 7200. Only valid for services configured to use load balancers."
```

Comment:
Add a `type`

Reply:
Yea, that's all true. =)

Different things slip through our Code Reviews depending on how much time we have or who is doing them. Generally, we prefer to be explicit rather than implicit, even if it's a bit verbose.

Reply:
I don't see any need for updating the readme as this variable is not mandatory and by default does not change the behaviour (+ I don't have make installed on my dev machine ;) Do you mind running the doc generator?